packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/overview/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Overview
+---
+Learn how to easily integrate biometric authentication like Face ID and Touch ID to your mobile apps that use Okta. 
+
+> Note: You must have already followed the [Sign users in to your mobile apps](/docs/guides/sign-into-mobile-app/android/before-you-begin/) guide to add biometric unlock with Touch ID, Face ID, and Smart Lock.
```

Comment:
Should we delete this? Two issues that I see:
1. Link is android specific
2. Referenced content doesn't have any information about Touch ID, Face ID and etc.

Reply:
Thanks for the catch on the link issue (need to remove the /android/before-you-begin/ part.

And no, it needs to be there, because it's a prerequisite for this guide. We are telling the user that they need to have already followed that guide before they can do the steps in this guide.